use-timescale/security/read-only-role.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Read-only role
+excerpt: Create a role to allow read-only access to your database
+product: [cloud, mst, self_hosted]
+keywords: [client credentials, accounts, users, members, read-only, security]
+tags: [authentication, credentials, members, security]
+---
+
+# Read-only user
+
+You can create a role that provides read-only access to your database.
+
+<Highlight type="important">
+Adding a read-only user role to your database does not provide resource
+isolation. If you require the security of a read-only user, as well as resource
+isolation, you can create a read replica instead. For more information, see the
+[read scaling](/use-timescale/latest/ha-replicas/read-scaling/)
+section.
+</Highlight>
+
+## Create a read-only user
+
+You can create a read-only user to provide limited access to your database.
+
+<Procedure>
+
+### Creating a read-only user
+
+1.  At the psql prompt, create the new role:
+
+    ```sql
+    CREATE ROLE readaccess;
+    ```
+
+1.  Grant the appropriate permissions for the role, as required:
+
+    ```sql
+    GRANT CONNECT ON DATABASE tsdb TO readaccess;
+    GRANT SELECT ON ALL TABLES IN SCHEMA public TO readaccess;
```

Comment:
`GRANT SELECT ON ALL TABLES IN SCHEMA public TO readaccess;` does not seem to work due to the `pg_buffercache` that `tsdbadmin` does not have access to. 
I will go to the developers and try to find a workaround.

Reply:
I figured it out. `pg_buffercache` can be dropped on the customer side using `tsdbadmin` and it is not automatically created on every service. Customers can decide whether to use `pg_buffercache` or not. 

Long story short, `GRANT SELECT ON ALL TABLES IN SCHEMA public TO readaccess;`  should work by default on all services. If anyone encounters this issue, the support team will be able to help them.